examples/hello_ocean.jl

```diff
@@ -0,0 +1,43 @@
+# # Hello, ocean!
+#
+# > How inappropriate to call this planet Earth, when it is quite clearly _Ocean_.
+#
+#   --Arthur C. Clark
+
+using Oceananigans, Oceananigans.Grids, Plots
```

Comment:
Would be ideal to not have `Oceananigans.Grids` in the introductory README example (same with `Oceananigans.Advection.WENO5`) but we already agreed to fix this in the future so it's good for now.